tests/platform_tests/conftest.py

```diff
@@ -71,7 +71,7 @@ def bring_up_dut_interfaces(request, duthosts, enum_rand_one_per_hwsku_frontend_
             duthost.no_shutdown(ifname=port)
 
 
-def get_state_times(timestamp, state, state_times):
+def get_state_times(timestamp, state, state_times, first_after_kexec=None):
```

Comment:
I did not understand well how this `first_after_kexec` helps for mac jump test case?

Reply:
I was calculating FDB ageing out events ONLY in the warm recovery path. That's why I was finding first expiry event after kexec, and ignore the first event in the going down path.

But, now I improved this in the latest commit. Now I measure all the FDB ageing out events between a time window.
This window  starts when SAI is instructed to disable MAC learning during warmboot shutdown path. The window ends when SAI is instructed to enable MAC learning (warmboot recovery path).

For this measurement I changed the `first_after_kexec` to `first_after_offset`, and set the offset to FDB-learning disable event.

There are FDB ageing out events in the collected log. But in pass case these events will occur ONLY after FDB learning is enabled back.

Reply:
For example, below data is from the test. Notice that `FDB_EVENT_OTHER_MAC_EXPIRY` starts after kexec and after FDB learning is disabled. If that happens before FDB learning is disabled, the test will fail.

```
"FDB_EVENT_OTHER_MAC_EXPIRY": {
	"timestamp": {
		"Start": "Nov 12 09:45:38.122770"
	},
	"Start count": 536,
	"time_taken": 516.558119,
	"last_occurence": "Nov 12 09:45:38.197544"
},
```
```		
"FDB_AGING_DISABLE": {
	"timestamp": {
		"Start": "Nov 12 09:36:06.577817",
		"End": "Nov 12 09:37:54.213523"
	},
	"start_time": null,
	"stop_time": null,
	"time_span": 107.635706
},
```
```
"reboot_time (kexec)": {
	"timestamp": {
		"Start": "Nov 12 09:37:01.564651"
	}
}
```